server/src/ThoughtspaceExtension.ts

```diff
@@ -64,11 +64,9 @@ const log = (...args: [...any, ...([ConsoleMethod] | [])]) => {
   const lastArg = args[args.length - 1]
   if (typeof lastArg === 'object' && Object.keys(lastArg).length === 1 && lastArg.method) {
     args = args.slice(0, -1)
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     method = lastArg.method
   }
-
-  // eslint-disable-next-line no-console
-  console[method](...args)
```

Comment:
Removed this by accident?

Reply:
In this case `console[method](...args)` is actual application logic. It's part of a custom logging method. You can't remove application logic, or it will change the behavior of the app.